lib/bosh/BoshDirectorClient.js

```diff
@@ -87,8 +202,8 @@ class BoshDirectorClient extends HttpClient {
           lockInfo.createdAt = new Date(lockInfo.createdAt);
           //Above check unnecessary, but for whatsoever reason if the lock is corrupted, we dont return back lockinfo
           const lockDuration = (new Date() - lockInfo.createdAt) / 1000;
-          logger.info(`Lock duration is : ${lockDuration} (secs) -- ${config.director.lock_deployment_max_duration}`);
-          if (lockDuration < config.director.lock_deployment_max_duration) {
+          logger.info(`Lock duration is : ${lockDuration} (secs) -- ${this.config.lock_deployment_max_duration}`);
+          if (lockDuration < this.config.lock_deployment_max_duration) {
```

Comment:
once `this.config` you can just refer `this.activePrimary[0].lock_deployment_max_duration` because all activePrimaries should have same config value for this.